### DISCUSSION TITLE
Hello,

### DISCUSSION THREAD

**Discussion by EnesGrc2** [IBAX-io/go-ibax#466] [Q&A] [answered]
Hello,
Yesterday i created a wallet and saved my private key as well as the password. Later my friend told me i was supposed to save back-up keys too. I didn't know it. Normally we have a private key i mean seed and the password. I tried to log into my wallet, i clicked import and entered my key (15 words), password. Unfortunately, i can not log in.  that's the error i receive, i hope i could uploaded properly
![image](https://user-images.githubusercontent.com/97418613/148710836-c473ca12-080c-4f70-9f67-6e7b1a105e84.png)
provided key is corrupted or contains invalid data.

i have been struggling with this for more than 6 hours. i referred my friends as well, don't want to ruin it.
How can i import my wallet, why my seed doesn't work. i am %100 sure password and my seed is right. The problem is i didn't know i will need back up seed. Could you help me please?
Thank you

_Originally posted by @furkangencer4 in https://github.com/IBAX-io/go-ibax/discussions/33_

**Comment by Yavuz101** [comment]
✌

**Comment by HalloKyle** [comment, marked as answer]
Hi
Please check this Link for sulution:
https://github.com/IBAX-io/go-ibax/discussions/62
